nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -147,6 +147,44 @@ $(CsWinRTFilters)
     <Copy SourceFiles="$(CsWinRTGeneratedFilesDir)\$(AssemblyName).winmd" DestinationFolder="$(TargetDir)" UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
   </Target>
 
+  <!-- ====================== -->
+  <!-- Call The GuidPatch tool on the projection .dll -->
+  <Target Name="CsWinRTInvokeGuidPatcher" Condition="'$(CsWinRTEnableGuidPatch)' == 'true'" AfterTargets="Compile" BeforeTargets="Link">
+    <!-- We need to invoke GuidPatch.exe. This lives in two different places depending on our environment. 
+
+         If we are in the CsWinRT repo, we don't have a .nupkg to use to find the exe, 
+           then we need to rely on CsWinRTGuidPatchRepoPath and CsWinRTGuidPatchRuntimePath from src/Directory.Build.props 
+           to get a hold of the files needed 
+           
+         If we are not in the CsWinRT repo, we can use the package paths we know -->
+
+    <PropertyGroup Condition="'$(CsWinRTInRepo)' == 'true'">
+      <!-- We use _DirectoryBuildTargetsBasePath as it is the only way I found to get "c:/.../cswinrt/src" -->
+      <CsWinRTGuidPatchExePath>$(_DirectoryBuildTargetsBasePath)\$(CsWinRTGuidPatchRepoPath)</CsWinRTGuidPatchExePath>
+      <CsWinRTGuidPatchRuntimeDll>$(_DirectoryBuildTargetsBasePath)\$(CsWinRTGuidPatchRuntimePath)</CsWinRTGuidPatchRuntimeDll> 
+    </PropertyGroup>
+
+    <PropertyGroup Condition="'$(CsWinRTInRepo)' != 'true'">
+      <CsWinRTGuidPatchExePath>$(CsWinRTPath)build\tools\</CsWinRTGuidPatchExePath>
+      <CsWinRTGuidPatchRuntimeDll>$(CsWinRTPath)lib\net5.0\</CsWinRTGuidPatchRuntimeDll>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <CsWinRTGuidPatchInput Condition="'$(CsWinRTGuidPatchInput)' == ''">@(BuiltProjectOutputGroupKeyOutput->'%(Identity)')</CsWinRTGuidPatchInput>
+    </PropertyGroup>
+
+    <Exec Command="$(CsWinRTGuidPatchExePath)GuidPatch.exe $(CsWinRTGuidPatchInput) $(CsWinRTGuidPatchRuntimeDll)" ConsoleToMsBuild="true">
+      <Output TaskParameter="ConsoleOutput" PropertyName="CsWinRTGuidPatchOutput" />
+    </Exec>
+    
+    <ItemGroup>
+      <PatchedFiles Include="$(_DirectoryBuildTargetsBasePath)\GuidPatcherOutput\*.dll" />
```

Comment:
Is this always set even you don't use directory.builds.targets?

Reply:
Yeah for now I really should follow my pattern of casing on the repo. Will follow up offline